src/DevBetterWeb.Core/Entities/Member.cs

```diff
@@ -66,7 +66,8 @@ internal Member(string userId, string firstName, string lastName)
   public decimal? CityLatitude { get; set; }
   public decimal? CityLongitude { get; set; }
   public List<BillingActivity> BillingActivities { get; set; } = new List<BillingActivity>();
-  //public List<MemberVideoProgress> Videos { get; private set; } = new List<MemberVideoProgress>();
+  public List<MemberFavoriteArchiveVideo> FavoriteArchiveVideos { get; set; } = new List<MemberFavoriteArchiveVideo>();
+  public List<MemberVideoProgress> Videos { get; private set; } = new List<MemberVideoProgress>();
```

Comment:
Same feedback here regarding encapsulation.

Also if we're using the aggregate design pattern (which we don't seem to be but probably should be) then you'd want to pick from one side or the other to be the "owner" of favorites. That is, either a member adds a favorite video, or a video adds a favoriting member, but not both. This help ensure there is only one place where you need to perform validation on this operation, among other things.